src/components/common/ColonyActionsTable/partials/ActionBadge/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './ActionBadge.tsx';
```

Comment:
Please remove the index file as we're moving away from that pattern to avoid having a large number of barrel files